lib/storage-handler.js

```diff
@@ -253,6 +253,7 @@ exports.download = function(provider, req, res, container, file, cb) {
     reader.pipe(res);
 
     reader.on('error', function onReaderError(err) {
+      res.set('content-type', 'application/json');
```

Comment:
> lb 2.x
> then in rest-adapter strong-remoting@2.x it only does a res.send()

That looks like a bug in strong-remoting's default error handler. Can we fix strong-remoting instead of working around the problem here?